src/commands/context/spec/context.spec.server.ts

```diff
@@ -90,7 +90,7 @@ describe('sasjs context', () => {
 
         expect(contexts.length).toBeGreaterThan(0)
       },
-      60 * 4 * 1000
+      10 * 60 * 1000
```

Comment:
You can just remove the custom timeout and use the default.

Reply:
fixed